sync/class.jetpack-sync-module-full-sync.php

```diff
@@ -69,6 +69,10 @@ function start( $modules = null ) {
 			$modules = array();
 		}
 
+		if ( isset( $modules['users'] ) && $modules['users'] === 'initial' ) {
+			$modules['users'] = Jetpack_Sync_Actions::get_initial_sync_user_config();
```

Comment:
This `get_initial_sync_user_config()` function should probably be in the users full sync module. Can do in a separate PR.